examples/scripts/ct_abel_tv_admm_tune.py

```diff
@@ -82,8 +81,8 @@ def setup(self, config, x_gt, x0, y):
         this case). The remaining parameters are objects that are passed
         to the evaluation function via the ray object store.
         """
-        # Put main arrays on jax device.
-        self.x_gt, self.x0, self.y = jax.device_put([x_gt, x0, y])
```

Comment:
Why remove `device_put` (here and in other scripts)? I read that as "put these matrices on my GPU if I have one" and that seems perfectly reasonable (and orthogonal to the `ensure_on_device` issue).

Reply:
It was replaced with conversion to a jax array using `jnp.array` or `snp.array` in the other scripts since I think that more closely matches what we want (ensure it's a jax array rather than more explicitly put it on a "device"), but this specific example may have been overealous.